docs/proposals/source-verificiation-policies.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Source Verification Policies
+authors:
+  - "@jannfis" # Authors' github accounts here.
+sponsors:
+  - TBD        # List all interested parties here.
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+creation-date: 2023-08-08
+last-updated: 2023-08-08
+---
+
+# Source Verification Policies
+
+This proposal introduces an evolution to the existing GnuPG commit verification feature.
+
+## Open Questions [optional]
+
+* Do we really need `progressive` verification level (see below)?
+* Should repository pattern matching be regexp based instead of shell-glob?
```

Comment:
Most stuff in AppProject is glob. And I think source templates are also defined with globs. So I'd favor that strategy until we have a strong reason to move to regex.